odh/base/monitoring/github-receiver.yaml

```diff
@@ -29,7 +29,8 @@ spec:
                  "--org=operate-first",
                  "--repo=SRE",
                  "--label=in progress",
-                 "--enable-inmemory=false"]
+                 "--enable-inmemory=false",
+                 "--enable-auto-close=true"]
```

Comment:
Can we make this `args` property into a  block style YAML sequence? Like:
```yaml
- --authtoken=...
- --repo=SRE
- --label="in progress"
...
```
It'll make it a bit more maintainable + easier to add values in future (you're not modifying the values around, since you don't need the `,` and `[]`)

Reply:
@tumido updated :+1: